dask/dataframe/backends.py

```diff
@@ -45,6 +46,50 @@
 from dask.sizeof import SimpleSizeof, sizeof
 from dask.utils import is_arraylike, is_series_like, typename
 
+
+class DataFrameBackendEntrypoint(DaskBackendEntrypoint):
+    """Dask-DataFrame version of ``DaskBackendEntrypoint``"""
+
+    def __init__(self):
+        """Register data-directed dispatch functions"""
+        raise NotImplementedError
+
+    @staticmethod
+    def from_dict(*args, **kwargs):
```

Comment:
So, the default `from_dict` has
https://github.com/dask/dask/blob/f382d9e8439f5ed200da6ce66df7b2b33a0fc500/dask/dataframe/core.py#L6039-L6042

That actually seems okay to me. Is there a downside to simply copying that, and adding on a `**kwargs` to the end to allow for backend-specific arguments?

I don't think it's appropriate to have the full zoo of parameters that something like `read_csv` or `read_parquet` have, but something like a minimum-viable-API seems like a good idea to me.